server.R

```diff
@@ -35,8 +35,9 @@ shinyServer(function(input, output, session) {
   WEEKS_IN_YEAR <- 52L
 
   load_parameters_data <- reactive({
+
     show_all_parameters(
-      input$parameters_SQ$datapath,
+      input$parameters_SQ,
       input$parameters_Reform$datapath
```

Comment:
At first glance I'm not sure why you remove "$datapath" from input$parameters_SQ but not from input$parameters_Reform, maybe it will make sense as I go.

Reply:
Now the input for the SQ parameter files are sourced from the "inst" folder using the selectInput() function, instead of fileInput() before. Although, now, this app has a limitation that we cannot compare two non-SQ policy settings. The SQ input has to be the SQ parameter from the internal files. I will leave it to your guys to decide on it.